components/org.wso2.carbon.identity.handler.event.account.lock/src/main/java/org/wso2/carbon/identity/handler/event/account/lock/util/AccountUtil.java

```diff
@@ -145,4 +149,41 @@ public static boolean isTemplateExists(String templateType, String tenantDomain)
                             + "tenantDomain: " + tenantDomain, e);
         }
     }
+
+    /**
+     * Publishes an event.
+     *
+     * @param eventName                 Event name
+     * @param properties                Event properties
+     * @throws AccountLockException     if event sent failed
+     */
+
+    public static void publishEvent(String eventName, Map<String, Object> properties) throws AccountLockException {
+
+        Event identityMgtEvent = new Event(eventName, properties);
+        try {
+            AccountServiceDataHolder.getInstance().getIdentityEventService().handleEvent(identityMgtEvent);
+        } catch (IdentityEventException e) {
+            String errorMsg = "Error occurred while triggering the event : " + identityMgtEvent.getEventName();
+            throw new AccountLockException(errorMsg, e);
+        }
+    }
+
+    /**
+     * Clones the given map.
+     *
+     * @param map          Map.
+     * @return             Cloned Map.
+     */
+    
```

Comment:
unnecessary new line

Reply:
Fixed in 5850c16fba894cba62d1f70e77d3088bc726777a